src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -187,6 +214,6 @@ private function getOriginalToken(TokenInterface $token)
             }
         }
 
-        return false;
+        return;
```

Comment:
This return statement should be removed (that's what we're doing everywhere else).